### PR TITLE
MS-1459  task: update README.md

### PR DESCRIPTION
**Pull request by jjrobothamho**
- update README description

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
-# modern-slavery-data-service
-Provides the service to store and manage the data within the modern slavery application
+# Modern Slavery Data Service
+
+Provides the service to store and manage the data within the <a href="https://github.com/UKHomeOffice/modern-slavery" target="_blank"> modern slavery</a> application.
```
Comment (zuzak) [inline README.md]:
```suggestion
Provides the service to store and manage the data within the [modern slavery](https://github.com/UKHomeOffice/modern-slavery) application.
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,8 @@
-# modern-slavery-data-service
-Provides the service to store and manage the data within the modern slavery application
+# Modern Slavery Data Service
+Provides a service layer to talk to a database to store and manage data for the <a href="https://github.com/UKHomeOffice/modern-slavery" target="_blank"> modern slavery</a> application
+
+The main modern slavery service has the ability to save an application or read a saved application. In order to do this, it needs to use this modern-slavery-data-service.
+
+**Saving:** The main application will send the data to this service as an end point.  This service will connect to a database and store that data and respond back to the main application if successful
+
+**Reading:** The main application will make a request to this modern-slavery-data-service.  This service will then connect to the database, get the data back and send it to the main application if it is successful
```
Comment (sulthan-ahmed) [inline README.md]: why is it all italic here?

Comment (zuzak) [reply README.md]: The HTML's probably confusing it. Change the link syntax to be [markdown-native](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links) and it'll probably resolve itself

Comment (sulthan-ahmed) [reply README.md]: 👍 worked a charm, thank you

**Pull request closed without merging**